newlib/libc/tinystdio/vfprintf.c

```diff
@@ -886,8 +886,12 @@ int vfprintf (FILE * stream, const char *fmt, va_list ap)
 }
 
 #ifndef vfprintf
+#if defined(__MACH__)
```

Comment:
Could this use #ifdef HAVE_ALIAS_ATTRIBUTE instead? Does this work?

#ifndef vfprintf
#ifdef HAVE_ALIAS_ATTRIBUTE
__strong_reference(vfprintf, __d_vfprintf);
#else
int __d_vfprintf (FILE * stream, const char *fmt, va_list ap) { return vfprintf(stream, fmt, ap); }
#endif
#endif

Reply:
That seems to work on my machine, will replace the commit